netutil/freeport.go

```diff
@@ -0,0 +1,41 @@
+package netutil
+
+import (
+	"net"
+)
+
+// FreeTCPAddr returns an available TCP port
+func FreeTCPAddr() *net.TCPAddr {
+	addr, err := net.ResolveTCPAddr("tcp", "localhost:0")
+	if err != nil {
+		panic(err)
```

Comment:
These methods are actually used as test helpers within bsk, so I would either make the package internal or return an err. What do you think would be the best approach? @socialpoint-labs/collaborators